common/spec/dependabot/config/update_config_spec.rb

```diff
@@ -148,6 +148,26 @@
         end
       end
     end
+
+    context "when the dependency version isn't known" do
+      let(:dependency) do
+        Dependabot::Dependency.new(
+          name: "actions/checkout",
+          requirements: [],
+          version: nil,
+          package_manager: "go_modules"
```

Comment:
```suggestion
          package_manager: "github_actions"
```

Doesn't really matter but we could make this match the dependency name unless it specifically needs to be `go_modules`?